blockstore/apps/api/v1/tests/test_contract.py

```diff
@@ -390,25 +385,87 @@ def test_editing_draft(self):
-
-    We need this because response.data is too smart about deserializing
-    (e.g. it will automatically parse UUID strings into a UUID object.), and
-    we want the basic primitives exactly as the REST API returns them so
-    that we can be more confident about compatibility. For instance, UUIDs
-    can be represented in multiple ways and parse the same, but if we
-    suddenly change the output format, we've broken backwards compatibility.
-    """
-    return json.loads(response.content.decode('utf-8'))
+        self.library_bundle_data = create_bundle_with_history(
+            self.client,
+            self.library_collection_data['uuid'],
+            "Dogs Library Bundle 🐶",
+            [
+                {'dog.txt': encode_str_for_draft("Rusty! 🐕")},
+                {'dog.txt': encode_str_for_draft("Jack! 🐕")},
+                {'dog.txt': encode_str_for_draft("Clyde! 🐕")},
+            ]
+        )
+        self.course_bundle_data = create_bundle_with_history(
+            self.client,
+            self.course_collection_data['uuid'],
+            "Course Bundle",
+            [
+                {
+                    'overview.md': encode_str_for_draft(
+                        "## TODO: 🤫 Make this an RST file instead!"
+                    )
+                }
+            ]
+        )
+
+    def test_simple_links(self):
+        """
+        Links: Create (in Draft), Commit, Delete
+        """
+        course_draft_url = self.course_bundle_data['drafts']['test_draft']
+        new_link_data = {
+            "dog_library": {
+                "bundle_uuid": self.library_bundle_data['uuid'],
+                "version": 3  # Use the latest -- TODO: should we default to latest?
+            }
+        }
+        self.client.patch(
+            course_draft_url, data={'links': new_link_data}, format='json',
+        )
+        draft_data = response_data(self.client.get(course_draft_url))
+        draft_link_data = draft_data['staged_draft']['links']['dog_library']
+
+        assert draft_link_data['modified'] is True
+        assert draft_link_data['direct']['bundle_uuid'] == self.library_bundle_data['uuid']
+        assert draft_link_data['direct']['version'] == 3
+        assert draft_link_data['direct'].get('snapshot_digest') is not None
+        assert draft_link_data['indirect'] == []
+
+        commit_resp_data = response_data(
+            self.client.post(course_draft_url + '/commit')
+        )
+        new_bv_url = commit_resp_data['bundle_version']
+        new_bv_data = response_data(self.client.get(new_bv_url))
+        bv_link_data = new_bv_data['snapshot']['links']['dog_library']
+
+        # The link data should be identical except that only Drafts have a
+        # 'modified' key...
+        draft_link_data.pop('modified')
+        assert draft_link_data == bv_link_data
+
+        self.client.patch(
+            course_draft_url, data={'links': {'dog_library': None}}, format='json'
```

Comment:
Shouldn't there be some code after this to check that the specified link was deleted?